cfme/fixtures/xunit_tools.py

```diff
@@ -29,7 +29,7 @@
 compiled_blacklist = re.compile('(' + ')|('.join(blacklist) + ')')
 
 
-timestamp = '{:%Y%m%d%H%M%S}'.format(datetime.datetime.now())
```

Comment:
consider calling the `.strftime` method or a call to the `format` for readability here

Reply:
done